parkings/api/enforcement/valid_parking.py

```diff
@@ -136,6 +136,9 @@ def _get_filterset(self, queryset):
         filter_backend = self.filter_backends[0]()
         return filter_backend.get_filterset(self.request, queryset, self)
 
+    def get_queryset(self):
+        return super().get_queryset().filter(domain=self.request.user.enforcer.enforced_domain)
```

Comment:
Add a separate commit before this commit for `IsEnforcer` permission class and putting it into use for all Enforcement API endpoints, since otherwise we cannot be sure that  `request.user.enforcer` exists.